### PR TITLE
squashed refactoring

### PR DESCRIPTION
**Pull request by unrealwork**
## Refactor test
- Created abstract SqlTest class with reused metods
- All tests inherited from SqlTest

## Review tasks
- [x] Move this logic of `sendSamplesToSeries` method to SqlMethod class

### CODE REVIEW

**Comment by KutuzovGregory** [comment]
seems ok

**Comment by rmakulov** [comment]
Ok

File: src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import java.util.Arrays;
+
+/**
+ * @author Igor Shmagrinskiy
+ */
+public abstract class SqlTest extends SqlMethod {
+
+
+    protected static void sendSamplesToSeries(Series series, Sample... samples) {
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java]: Move this logic to SqlMethod class

**Comment by Sild** [comment]
OK

**Comment by KutuzovGregory** [comment]
Ok

**Comment by rmakulov** [comment]
ok

**Pull request merged**